Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathProjection.php

```diff
@@ -657,7 +658,7 @@ private function moveNode(
     ): void {
         $this->disconnectNodeFromSiblings($node);
 
-        $this->connectNodeWithSiblings($node, $newParentNodeAggregateId, $newSucceedingNodeAggregateId);
+        $this->connectNodeWithSiblings($node, $newParentNodeAggregateId, $newSucceedingNodeAggregateId, true);
```

Comment:
@nezaniel I can change this to 
```suggestion
        $this->connectNodeWithSiblings($node, $newParentNodeAggregateId, $newSucceedingNodeAggregateId, false);
```
without any tests failing..
That makes me wonder whether we could just remove the `$includingSelf` parameter below

Reply:
mmh, weird... the projection looked really funny (and not in a good way) when I omitted this, will re-check and add failing tests

Reply:
I withdraw my point and claim the exact opposite: You're correct, seems like I fixed the weirdness along the way by accident... 

I removed the switch, looks a lot better now.